apps/openmw/mwworld/worldimp.cpp

```diff
@@ -1187,9 +1190,8 @@ namespace MWWorld
                     mWorldScene->addObjectToScene(newPtr);
 
                     std::string script = newPtr.getClass().getScript(newPtr);
-                    if (!script.empty()) {
+                    if (!script.empty())
                         mLocalScripts.add(script, newPtr);
-                    }
```

Comment:
Is this in our style-guideline? 

I'm OK with have redundant {} since it doesn't hurt to have them anyway. For me it is easier to read in general.

Reply:
Ehh. Reverted.